src/main/java/de/wi2020sebgroup1/cinema/entities/User.java

```diff
@@ -55,6 +55,16 @@ public User(String userName, String name, String firstName, String email, String
 		this.password = password;
 		this.role = role;
 	}
+	
+	public User(String userName, String name, String firstName, String email, String password) {
+		super();
+		this.userName = userName;
+		this.name = name;
+		this.firstName = firstName;
+		this.email = email;
+		this.password = password;
+		this.role = new Role("Default User", "user");
```

Comment:
Vorerst okay, muss (Sobald existent) durch die Standardrolle in der DB ersetzt werden ^^.